docs/source/protocol.rst

```diff
@@ -120,33 +119,21 @@ Auxiliary Payload
 - See: https://developer.gnome.org/glib/stable/gvariant-format-strings.html#gvariant-format-strings-maybe-types
 - Details for each event ID listed in :ref:`events page`
 
-Total format
-++++++++++++
-
-In total should look like ``(uayxmv)``.
-
 Aggregate Metric
 ~~~~~~~~~~~~~~~~
 
 Aggregate metrics indicate counts that summarize a value of interest (e.g., a
 very common event happening n times in a particular time interval or
-fluctuations in heap size over time). Counts can be positive, zero, or
-negative. They are identical to the singular metrics but have an added counter
-field in the wire format.
+fluctuations in heap size over time). Counts are always strictly positive. They
+are identical to the singular metrics but have an added counter field in the
```

Comment:
They're not identical to singular metrics any more – occured_at becomes a yyyy-mm-dd interval start.

Reply:
Right. I’ve added a paragraph about that.